### PR TITLE
feature: adds umd example for vanilla html/js

### PR DESCRIPTION
**Pull request by awalias**
adds simple UMD html/js auth example

### CODE REVIEW

**Comment by awalias** [comment]
image

**Comment by kiwicopple** [comment]
good to see the `awalias` pink branding on this one 💅 

Nice one thanks Ant - looks good to me after resolving some conflicts. Feel free to merge whenever

File: examples/auth-vanilla-js/package.json [hunk 1]
```diff
@@ -4,16 +4,8 @@
...
+    "dev": "npx light-server -s ."
   },
   "author": "",
-  "license": "ISC",
-  "dependencies": {
-    "@supabase/supabase-js": "^0.34.5",
-    "webpack": "^4.43.0",
-    "webpack-cli": "^3.3.12"
-  }
+  "license": "ISC"
```
Comment (kiwicopple) [inline examples/auth-vanilla-js/package.json:10]: MIT?

File: examples/auth-vanilla-js/README.md [hunk 2]
```diff
@@ -1,27 +1,12 @@
 # Vanilla-js Auth Example
 
-how to sign up and login using supabase and supabase-js
+how to sign up and login using supabase and supabase-js using HTML and Javascript only
```
Comment (kiwicopple) [inline examples/auth-vanilla-js/README.md:3]: how = "How"?

**Pull request merged**